tests/DeltaLake.Tests/Settings.cs

```diff
@@ -2,6 +2,6 @@ namespace DeltaLake.Tests
 {
     public static class Settings
     {
-        public static string TestRoot = System.Environment.GetEnvironmentVariable("TEST_ROOT") ?? Path.Join("..", "..", "..", "..", "data");
+        public static string TestRoot = System.Environment.GetEnvironmentVariable("TEST_ROOT") ?? Path.Combine("..", "..", "..", "..", "data");

```

Comment:
(The differences are in their platform availability and handling of null and empty arguments).